packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx

```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
+                  online video sharing. And we're only getting started.
+                </Text>
+              </HeaderGridItem>
+            </CenteredLayoutGrid>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 1, lg: 2 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer>
+                  <CardImage dropShadow absolute src={dashboardImgBack} alt="Dashboard" />
+                  <CardImage dropShadow src={dashboardImgFront} ref={dashboardImageRef} alt="Reward card" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  title="Manage your channel on a simple dashboard"
+                  subtitle="YPP dashboard"
+                  body="There you can check your channel status and get instructions on completing all actions to get rewards"
+                />
+              </GridItem>
+            </CardImageRow>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 6, lg: 6 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer positionOnMobile="flex-end">
+                  <CardImage absolute dropShadow src={myVideosBack} alt="My videos" />
+                  <CardImage dropShadow src={myVideosFront} ref={myVideosImageRef} alt="Button" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem
+                colStart={{ sm: 3, md: 1, lg: 2 }}
+                rowStart={{ md: 1 }}
+                colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}
+              >
+                <ContentCard
+                  title="Backup your videos and own them forever"
+                  subtitle="YouTube backup"
+                  body="You can sleep peacefully knowing that your conent is safe and forever stored on blockchain. You can even sell it as NFTs — you own it forever."
+                />
+              </GridItem>
+            </CardImageRow>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 1, lg: 2 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer positionOnMobile="center">
+                  <CardImage absolute src={youtubeSyncBack} alt="Video tiles" />
+                  <CardImage src={youtubeSyncFront} ref={youtubeSyncImageRef} alt="Video tile" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  pill={{
+                    label: 'Coming later this year',
+                  }}
+                  title="Automatic YouTube video sync"
+                  subtitle="YouTube sync"
+                  body="Set up once and do nothing. Your videos will be automaticly synced from youtube to Atlas and you will still earn for it passively."
```

Comment:
In some places we write Youtube with a capital letter and sometimes we don't, probably to consult with designers.

Reply:
In [this frame](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1606%3A85515) only `YouTube` is used. I don't think it's worth consulting this :P